beam/www/beam/pages/WorkOrder.vue

```diff
@@ -2,14 +2,15 @@
 	<Navbar>
 		<template #title>
 			<h1 class="nav-title">Manufacture</h1>
+			<span v-if="store.form.dirty" class="nav-subtitle">Unsaved changes</span>
```

Comment:
@agritheory I've added an empty `nav-subtitle` class reference in case this needs to be styled. Currently, if any changes are made to the form, the page would look something like this:

![image](https://github.com/user-attachments/assets/a8372ee0-6dd0-485c-b32a-4d847583788f)